usr/src/cmd/zdb/zdb.c

```diff
@@ -646,26 +647,26 @@ get_dtl_refcount(vdev_t *vd)
 		return (0);
 	}
 
-	for (int c = 0; c < vd->vdev_children; c++)
+	for (unsigned c = 0; c < vd->vdev_children; c++)
 		refcount += get_dtl_refcount(vd->vdev_child[c]);
 	return (refcount);
 }
 
-int
+static int
 get_metaslab_refcount(vdev_t *vd)
 {
 	int refcount = 0;
 
 	if (vd->vdev_top == vd && !vd->vdev_removing) {
-		for (int m = 0; m < vd->vdev_ms_count; m++) {
+		for (unsigned int m = 0; m < vd->vdev_ms_count; m++) {
```

Comment:
I agree, I'm just saying that unsigned integers should either all be declared as `unsigned int`, or all declared as simply `unsigned`.